app/src/main/java/org/gdg/frisbee/android/fragment/GdlListFragment.java

```diff
@@ -98,28 +85,25 @@ public void onActivityCreated(Bundle savedInstanceState) {
 
         mClient = new GoogleDevelopersLive();
 
-        mAdapter = new GdlAdapter(getActivity(), ((GdgActivity)getActivity()).getGoogleApiClient());
+        mAdapter = new GdlAdapter(getActivity(), ((GdgActivity) getActivity()).getGoogleApiClient());
         setListAdapter(mAdapter);
 
         setIsLoading(true);
 
         fetchContent(false);
 
-        if(getArguments().getBoolean("active")) {
+        if (getArguments().getBoolean("active")) {

```

Comment:
Yes, it should go to a separate PR. Just an observation. I'll raise new issue for that.